fast64_internal/oot/animation/operators.py

```diff
@@ -18,10 +18,15 @@
 
 
 def exportAnimationC(armatureObj: bpy.types.Object, settings: OOTAnimExportSettingsProperty):
+    if settings.isCustom:
+        checkEmptyName(settings.filename)
```

Comment:
shouldn't filename only be checked if isCustomFilename (which can be checked regardless of isCustom)